backend/app/store/image/image.go

```diff
@@ -101,6 +101,15 @@ func NewService(s Store, p ServiceParams) *Service {
 	return &Service{ServiceParams: p, store: s}
 }
 
+// SubmitAndCommit multiple ids immediately
+func (s *Service) SubmitAndCommit(idsFn func() []string) {
```

Comment:
this code inside a function now and can return an error, instead of logging. And whoever calls it could decide what to do with that error

Reply:
Should `Submit` return error too? Now `SubmitAndCommit` return error, but `Submit` not and it looks a bit inconsistent, especially here. https://github.com/umputun/remark42/pull/744/files#diff-0c9477a43c3f506684e6e23b4ce2e918R247-R256

But what error can return `Submit`, always nil for now?

Reply:
Submit just put things to a channel. As long as we have no context here to interrupt it I see no reason to return any error because it has no path returning any error.